developer-docs/about_rke2_docs.md

```diff
@@ -0,0 +1,31 @@
+# About RKE2 docs
+
+The RKE2 git repository currently holds two kinds of documentation:
+
+- RKE2 User Docs
+- RKE2 Developer Docs
+
+There are other differences than their audiences, and this document elaborates on it.
+
+## User docs
+
+As its name suggests, the main target for this documentation is Rancher RKE2 users. Covering RKE2 topics such as architecture, installation and upgrade process, and security from an end-user perspective.
+
+The documentation's source lingers as markdown files in the `docs/` directory within this repository. These documents are processed using [mkdocs](https://www.mkdocs.org/) and served on <https://docs.rke2.io/> as a documentation website.
```

Comment:
> linger 
> stay in a place longer than necessary because of a reluctance to leave

Is this a commentary on our collective failure to move the docs out to a separate repo? If so I like it!

Reply:
Hehe, actually no, I wrote it by mere force of habit. Should I say "The documentation's source **are** markdown files [...]" or swap "linger" with something else?

Reply:
Maybe something like:
```suggestion
The documentation is contained in markdown files in the `docs/` directory within this repository. These documents are processed using [mkdocs](https://www.mkdocs.org/) and served on <https://docs.rke2.io/> as a documentation website.
```